src/scene-entry-manager.js

```diff
@@ -155,7 +159,7 @@ export default class SceneEntryManager {
     const { avatarId, displayName } = this.store.state.profile;
     this.playerRig.setAttribute("player-info", {
       displayName,
-      avatarSrc: avatarId && avatarId.startsWith("http") ? proxiedUrlFor(avatarId) : `#${avatarId || "botdefault"}`
+      avatarSrc: getAvatarSrc(avatarId, avatarCachebuster++)
```

Comment:
surprised by this -- we shouldn't be HTTP caching the reticulum URLs right?

Reply:
Yeah.. though we do GLTF cache it, and we want to broadcast the network update so that other clients fetch it, this does both, though I reworked how this is done since this was causing some issues.